src/cli/cli_mac_filter.cpp

```diff
@@ -218,6 +260,23 @@ template <> otError MacFilter::Process<Cmd("rss")>(Arg aArgs[])
             error = otLinkFilterAddRssIn(GetInstancePtr(), &extAddr, rss);
         }
     }
+    /**
+     * @cli macfilter rss add
+     * @code
+     * macfilter rss add * -50
+     * Done
+     * @endcode
+     * @code
+     * macfilter rss add 0f6127e33af6b404 -85
+     * Done
+     * @endcode
+     * @cparam macfilter rss add @ca{extaddr} @ca{rss}
+     * @par api_copy
+     * #otLinkFilterAddRssIn
+     * @par
+     * To set a default value for the received signal strength for all received messages,
+     * use the `*` for the `extaddr` argument, as shown in the example.
```

Comment:
Let's remove "as shown in the example" everywhere you've used it in this PR.  We typically don't use that wording in the CLI docs.  They all have examples.